src/sidebar/components/test/AnnotationView-test.tsx

```diff
@@ -1,27 +1,44 @@
 import { mount } from 'enzyme';
+import sinon from 'sinon';
+import type { SinonStub } from 'sinon';
 
 import { mockImportedComponents } from '../../../test-util/mock-imported-components';
+import { getImports } from '../../../test-util/mockable-imports';
 import { waitFor } from '../../../test-util/wait';
-import AnnotationView, { $imports } from '../AnnotationView';
+import type { LoadAnnotationsService } from '../../services/load-annotations';
+import type { SidebarStore } from '../../store';
+import * as AnnotationViewExports from '../AnnotationView';
+import type { AnnotationViewProps } from '../AnnotationView';
+
+const AnnotationView = AnnotationViewExports.default;
+const $imports = getImports(AnnotationViewExports);
 
 describe('AnnotationView', () => {
-  let fakeStore;
-  let fakeOnLogin;
+  let highlightAnnotations: SinonStub;
+  let setExpanded: SinonStub;
+  let profile: SinonStub;
+  let fakeStore: SidebarStore;
+  let fakeOnLogin: SinonStub;
   let fakeUseRootThread;
-  let fakeLoadAnnotationsService;
+  let loadThread: SinonStub;
+  let fakeLoadAnnotationsService: LoadAnnotationsService;
 
   beforeEach(() => {
+    highlightAnnotations = sinon.stub();
+    setExpanded = sinon.stub();
+    profile = sinon.stub().returns({ userid: null });
```

Comment:
I extracted these so that they can be passed to the store mock, but then accessed directly on tests, ensuring proper type inference.